apps/judicial-system/backend/src/app/modules/case/limitedAccessCase.service.ts

```diff
@@ -540,23 +546,71 @@ export class LimitedAccessCaseService {
             { reason },
           ),
         )
+    })
+
+    await Promise.all(fileFetchPromises)
+
+    if (isRequestCase(theCase.type)) {
+      filesToZip.push(
+        {
+          data: await this.pdfService.getRequestPdf(theCase),
+          name: 'Krafa.pdf',
+        },
+        {
+          data: await this.pdfService.getCourtRecordPdf(theCase, user),
+          name: 'Þingbók.pdf',
+        },
+        {
+          data: await this.pdfService.getRulingPdf(theCase),
+          name: 'Úrskurður.pdf',
+        },
+      )
     }
 
-    filesToZip.push(
-      {
-        data: await this.pdfService.getRequestPdf(theCase),
-        name: 'krafa.pdf',
-      },
-      {
-        data: await this.pdfService.getCourtRecordPdf(theCase, user),
-        name: 'þingbok.pdf',
-      },
-      {
-        data: await this.pdfService.getRulingPdf(theCase),
-        name: 'urskurður.pdf',
-      },
-    )
+    if (isIndictmentCase(theCase.type)) {
+      const defendant = theCase.defendants && theCase.defendants[0]
```

Comment:
We are already sending subpoeanas for all defendants to the court and police. Even if no indictments are created at this time with more than one defendant it is easy to have the implementation handle multiple defendants. Please consider suppurting multiple defendants here so we don´t need to revisit this code when prosecutors start creating indictments with multiple defendants.